Makefile

```
@@ -270,3 +270,83 @@ start-i2p-full-env: .init-screen-session .start-i2p-seeds .start-i2p-clients .re
+	./scripts/desktop-ui-harness.bash click "$(selector)"
+
+desktop-ui-type:
+	./scripts/desktop-ui-harness.bash type "$(selector)" "$(text)"
+
+desktop-ui-wait-node:
+	./scripts/desktop-ui-harness.bash wait-node "$(selector)" "$(timeout_ms)" "$(visible)"
+
+desktop-ui-press-key:
+	./scripts/desktop-ui-harness.bash press-key "$(key)" "$(selector)"
+
+desktop-ui-scenario:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)